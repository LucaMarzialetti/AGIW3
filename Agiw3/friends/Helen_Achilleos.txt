--url_fb https://www.facebook.com/helen.achilleos.9
--friends
VM10848:3 Joley Richards
VM10848:3 Demetris Achilleos
VM10848:3 Donna Groves
VM10848:3 Victoria Noël
VM10848:3 Jo Carroll
VM10848:3 Dina Charalambous
VM10848:3 Anna Achilleos
VM10848:3 Anna Nicolaides
VM10848:3 Andy Fx
VM10848:3 Stefanie Yianni
VM10848:3 Antony Nicolaides
VM10848:3 Penny Savva
VM10848:3 Koulla Yianni
VM10848:3 Antoni Achilleos
VM10848:3 Mario Aza
VM10848:3 Anton John Nicolaides
VM10848:3 S Sunny DeoPaul
VM10848:3 Georgina Aza
VM10848:3 Kirsty Griffin
VM10848:3 Nick Yianni
VM10848:3 Dean Yianni
VM10848:3 Christiana Georgopoulos
VM10848:3 Alison O'Brien
VM10848:3 Mel Michael
VM10848:3 Skevoulla Zinonos
VM10848:3 Andrew Aza
VM10848:3 Petros Petrou
VM10848:3 Carol Demetriou
VM10848:3 Kyproulla Nicolaides
VM10848:3 Phil Achilleos
VM10848:3 Rafaella Nikolaidou
VM10848:3 Maria Nicolaides
VM10848:3 Michael Nicolaides
VM10848:3 Angela Cooper
VM10848:3 Mariam Michael
VM10848:3 Sandra OBrien
VM10848:3 Eleni Nicolaides
VM10848:3 Stavros Lion Buckmaster
VM10848:3 Maria Antoniou Kyriakoulis
VM10848:3 Grace Sheppard
VM10848:3 Ioanna Nikolaidou
VM10848:3 Angela Marjeram
VM10848:3 Peter Leandro Kypri
VM10848:3 Maria Thoma
VM10848:3 Dora Christodoulou
VM10848:3 Roula Petrou
VM10848:3 Colleen Bernadette Pound Byrne
VM10848:3 Theo Nicolaides
VM10848:3 Donna Stavy Smith
VM10848:3 Dino Achilleos
VM10848:3 Andria Georgiou
VM10848:3 John Wallace
VM10848:3 Robert Dwyer
VM10848:3 Hetty Dock
VM10848:3 Nicholas Yiacoumi
VM10848:3 Pambos Zenonos
VM10848:3 Elliot Juggins
VM10848:3 Ellie Charalambous
VM10848:3 Angela Stylianou
VM10848:3 Jayne Emily Easters
VM10848:3 Sherman Woolridge
VM10848:3 Kypros Nikolaides
VM10848:3 Sofia Km
VM10848:3 Naomi Gayle
VM10848:3 Peter Dwyer
VM10848:3 Serife Ozmindik
VM10848:3 Sansel Ozmindik
VM10848:3 Sotiris Alba
VM10848:3 Lucy Spacey
VM10848:3 Franco Barillari
VM10848:3 Caroline Onobrauche
VM10848:3 Dawn Johnson
VM10848:3 Anneka Demetriou
VM10848:3 Christina O'Reilly
VM10848:3 Kathy O'Brien
VM10848:3 Yihsiu Tsai
VM10848:3 Δημητράκης Δημητρίου
VM10848:3 Andriana Christakis
VM10848:3 Aggela Demetriou
VM10848:3 May Hughes Achilleos
VM10848:3 Little Chris
VM10848:3 Dimitris Achilleos
VM10848:3 Cara Demetriades
VM10848:3 Michael Mulholland
VM10848:3 Courtney Achilleos
VM10848:3 Stella Stylianou
VM10848:3 Robyn Chelsea Spencer
VM10848:3 Nadine Hartman Guzman
VM10848:3 Jay Achilleos
VM10848:3 Louise Godwin
VM10848:3 Melios Anastasiou
VM10848:3 Antria Damianou
VM10848:3 Hayriye Osman May
VM10848:3 Sarah Gooden
VM10848:3 William Murphy
VM10848:3 Margaret Baxter
VM10848:3 Lisa Ward
VM10848:3 Martyna Aite
VM10848:3 Antri Nicolaidou
VM10848:3 Maria Pikrodafnis
VM10848:3 Adrian Chase
VM10848:3 Al Youungy
VM10848:3 Daphne Perry
VM10848:3 Babs Chase
VM10848:3 Marios Achilleos
VM10848:3 Emma ADickinson
VM10848:3 Owen Palfrey
VM10848:3 Vasilios Bagalexis
VM10848:3 Petros Kyriakou
VM10848:3 Johnnys Nicolaides
VM10848:3 Rose Ahmet
VM10848:3 Paulos Mastoras
VM10848:3 Andreas Alba
VM10848:3 Kirsty Wicks
VM10848:3 Wendy James
VM10848:3 Michael Demetriou
VM10848:3 Maria Ioanna Omonoia
VM10848:3 Leanne French
VM10848:3 Mark Murphs
VM10848:3 Dimos Tsilis
VM10848:3 Barb Morken
VM10848:3 Maria Lazaridou
VM10848:3 Christina Stylianou
VM10848:3 Andy Fordyce-harvey
VM10848:3 Diana Nicolae
VM10848:3 Stephen James Demetriou
VM10848:3 Yu Kao
VM10848:3 Tyson Haugen
VM10848:3 Wapping Vineyard
VM10848:3 Theodora Achilleos
VM10848:3 Jeorgia Georgiou
VM10848:3 Marc De Beukeleer
VM10848:3 Tania Nassou
VM10848:3 Maria Lazaridou
VM10848:3 Reece Coachp
VM10848:3 Re Master
VM10848:3 Joseph Pelino
VM10848:3 Chantelle Leah Jones
VM10848:3 Frixos Matas GateOne
VM10848:3 Gano Excel Cyprus
VM10848:3 ΖΕΦΗ ΖΡΑΦΝΟΠΟΥΛΟΥ
VM10848:3 Serti Catarina
VM10848:3 Renee Leggett
VM10848:3 Katrine Alequin DelValle
VM10848:3 Jose Maria Fernandez
VM10848:3 Maria Tsilochristou
VM10848:3 Jordan Haugen
VM10848:3 Teresa Teodoro
VM10848:3 Fernanda Mirante
VM10848:3 Giannis Xousiadis
VM10848:3 Carla Queiroz
VM10848:3 Amanda Wilson
VM10848:3 Giannos Thoma
VM10848:3 Jessica O'connor
VM10848:3 Brian Nelson
VM10848:3 Lynn Soehnel Braun
VM10848:3 Jana Carron Moore
VM10848:3 Rodolfo Gomes
VM10848:3 Foulla Nikolaidou
VM10848:3 Susete Sousa
VM10848:3 Tracie Dirtystopout Flynn
VM10848:3 Antigoni Demetriades
VM10848:3 John Nicolaides
VM10848:3 Cathy Miller
VM10848:3 Damon Sweet
VM10848:3 Andrea Atkin
VM10848:3 Phillip Hamilton
VM10848:3 Venoyd McCalister III
VM10848:3 Lori Hartman
VM10848:3 Shannen Murphy
VM10848:3 Nita Coronado-bom
VM10848:3 Ilda Rivero Venta
VM10848:3 Dimitris Glymidakis
VM10848:3 Maria Chaplin
VM10848:3 Abdurrahım Secer
VM10848:3 Sofia Fitneogloy
VM10848:3 Sandy Mo
VM10848:3 Dianne Shelton
VM10848:3 German Guzman
VM10848:3 Michelle Moore Monaghan
VM10848:3 Irw Tsilh
VM10848:3 Lenka Francova
VM10848:3 Liz Bamborough
VM10848:3 Andy Demetriades
VM10848:3 Dread Lock Farmer
VM10848:3 Hanna Gustavsson
VM10848:3 Adelina Okros
VM10848:3 Niki Arfani
VM10848:3 Charo Trabado
VM10848:3 Tommy Griffin
VM10848:3 Pietie Hoekstra
VM10848:3 Methis Zenonos
VM10848:3 Sophia Glegle
VM10848:3 Nour Moukdad
VM10848:3 Farm Aggeliki
VM10848:3 Elena Stoimenova
VM10848:3 Maha Awad
VM10848:3 Kathy Hall
VM10848:3 Maria Vassilliou
VM10848:3 Sabrina Leggett
VM10848:3 Mert Suzie Evsal Tunççağ
VM10848:3 Bro Heinz
VM10848:3 Steven Andrew
VM10848:3 Johanna May Thirkettle
VM10848:3 Cynetradio Listen Live
VM10848:3 George Costa
VM10848:3 Julia Hine
VM10848:3 Aris Nikolaides
VM10848:3 Ioanna Nikolaidou
VM10848:3 Andreas Cooper
VM10848:3 Aris Niklaides
VM10848:3 Kirsty Millwall Wicks
VM10848:3 Margarita Nicolaidou
VM10848:3 Maria Nikolaidou
VM10848:3 Sam Bland
VM10848:3 Danielle Madgin
VM10848:3 Malcolm Murphy
VM10848:3 Feray Osman
VM10848:3 Jamie 'Milky' Moor
VM10848:3 James Cooper
VM10848:3 Kypros Nikolaides
VM10848:3 Charitini Petrou
VM10848:3 Antoni Achilleos
VM10848:3 Laura Cavill
VM10848:3 Alexia Andrea Georgopoulos
VM10848:3 Chrysis Achilleos
VM10848:3 Lucille Patton
VM10848:3 Cathrine Richards
VM10848:3 Yiannis Pikrodafnis
VM10848:3 Tony Georgiou
VM10848:3 Koulla Evdokia Lambrou
VM10848:3 Nadine Jackson
VM10848:3 George Demetriou
VM10848:3 Hannah Freeman
VM10848:3 Polymnia Pikrodafnis
VM10848:3 Jemma Paine
VM10848:3 Nicola Gordon
VM10848:3 Reece O'hara-Lewis
VM10848:3 Heather Palmer
VM10848:3 Raphael ThOmas
VM10848:3 Christiana Demetriou Theodorou
VM10848:3 Anthi Demetriou
VM10848:3 Harriet Simeon
VM10848:3 Helen Miller
VM10848:3 Lisa Shrubb
VM10848:3 Eleni Pikrodafnis
VM10848:3 Angelica Gray-Dally
VM10848:3 Joanne Banks
VM10848:3 Hung Tran
VM10848:3 Amber Townsend
VM10848:3 Karleen Bennett
VM10848:3 Cheryl Jones
VM10848:3 Angela MacDonald
VM10848:3 John Manesis
VM10848:3 Debbie Gokcimen
VM10848:3 Lynne Villiers
VM10848:3 Kypros Kypri
VM10848:3 Laura Hassain
VM10848:3 Cilla Goucher
VM10848:3 Temisan Neburagho