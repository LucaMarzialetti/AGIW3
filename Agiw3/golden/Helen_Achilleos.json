{
    "PATTERN": {
        "tel": [
            "0845 497 8160",
            "+44 161 831 8160",
            "0845 497 8888",
            "+44 161 831 8888"
        ],
        "addr": ["M1 5ES",
            "70 Great Bridgewater Street"],
        "email": []
    },
    "NER": {
        "LOC": ["Manchester",
            "United Kingdom"],
        "ORG": [
            "Eversheds House",
            "United Kingdom Construction",
            "United Kingdom Websites",
            "Thomson Reuters",
            "Facebook",
            "Google"
        ],
        "PER": [
            "Engineering Lawyer",
            "Helen Achilleos",
            "Helen Eversheds LLP Eversheds",
            "Attorney"
        ]
    },
    "url": "http://pview.findlaw.com/view/4763578_1"
}